packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md

```diff
@@ -3,11 +3,125 @@ id: Release highlights
+    - [@patternfly/react-log-viewer@6.1.0](https://www.npmjs.com/package/@patternfly/react-log-viewer) ([changelog](https://github.com/patternfly/react-log-viewer/releases/tag/v6.1.0))
+- [@patternfly/react-topology@6.2.0](https://www.npmjs.com/package/@patternfly/react-topology) ([changelog](https://github.com/patternfly/react-topology/releases/tag/v6.2.0))
+    - [@patternfly/react-user-feedback@6.1.0](https://www.npmjs.com/package/@patternfly/react-user-feedback) ([changelog](https://github.com/patternfly/react-user-feedback/releases/tag/v6.1.0))
+
+### PatternFly 6 enablement
+
+As products have continued their PatternFly 6 migration journeys, we've received helpful feedback on the upgrade process, including common roadblocks and situations that could use more guidance. So, we expanded and reorganized the [PatternFly 6 upgrade guide](/get-started/upgrade) to more clearly and comprehensively support the upgrade process. 
+
+### Component animations
+
+Our highly-anticipated animations project focuses on integrating motion design into PatternFly components, to enhance user experiences and provide additional cues for interaction. You can keep track of this project via [our micro-animations roadmap](https://github.com/orgs/patternfly/projects/16/views/23?pane=issue&itemId=86507128&issue=patternfly%7Cpf-roadmap%7C215). 
+
+Note that animations will either be specified as opt-in or opt-out:
+- **Opt-in:** Must be enabled in your code.
+- **Opt-out:** Turned on by default, can be disabled in your code.
+
+With this release, 3 components now support animations: 
+
+1. [Alert](/components/alert)   
+    - **Animation type:** Opt-in.
+    - **Animation behavior:** For alerts within a group, there's a slide-in animation as new alerts are received and a slide-out animation as alerts are dismissed.
+    - **Example:** To visualize the motion behavior, you can interact with our website examples, which have opted in to animations.  
+    - **Note:** By default, alert animations are opt-in and will only apply to alerts within an alert group. Opting into animations could require test updates. To ensure your test pass, the quickest solution is to set `hasAnimations` to `false`. Additionally, when alerts are dynamically added to a group, you must ensure that new alerts are prepended to the alert group list, rather than appended to the end of it.  
+1. [Navigation](/components/navigation) 
+    - **Animation type:** Opt-out.
+    - **Animation behavior:** For expandable navigation items, the sub-menu fades in and out as navigation items are expanded and collapsed.
+    - **Example:** As an opt-out animation, you can see this motion behavior by default across expandable navigation items.
+1. [Notification badge](/components/notification-badge)
+    - **Animation type:** Opt-in.
+    - **Animation behavior:** When a new notification arrives, the bell icon has a "ring" animation.
```

Comment:
Possibly more nitpicky, but something along the lines of "When the animation is triggered..." might be more accurate since It will be entirely up to the consumer to trigger the animation - though it most likely would be when a new notification arrives. My only worry is not wanting consumers to think we handle triggering the animation for them, even though our example does show it requires manual logic.